be/src/runtime/routine_load/data_consumer.cpp

```diff
@@ -444,6 +444,14 @@ Status PulsarDataConsumer::init(StreamLoadContext* ctx) {
         _custom_properties.emplace(item.first, item.second);
     }
 
+    std::string log_file_path = config::sys_log_dir + "/pulsar-cpp-client.log";
```

Comment:
Will this log file be cut and rolled by pulsar?
Otherwise, it will cause the file to become bigger and bigger all the time.

Reply:
It can't be rolled by pulsar, but it's better than printing logs into be.out.